library/ecp.c

```diff
@@ -1312,7 +1314,7 @@ static int ecp_normalize_jac( const mbedtls_ecp_group *grp, mbedtls_ecp_point *p
     if( mbedtls_internal_ecp_grp_capable( grp ) )
         return( mbedtls_internal_ecp_normalize_jac( grp, pt ) );
 #endif /* MBEDTLS_ECP_NORMALIZE_JAC_ALT */
-
+#if !defined(MBEDTLS_ECP_NO_FALLBACK) || !defined(MBEDTLS_ECP_NORMALIZE_JAC_ALT)
```

Comment:
I generally prefer to avoid double negations. As written in 97b4984657f2fc48dd128d2fd3a1ccf3628a7866, it makes sense to have the same condition for the variable declarations and for the bulk of the code. However, if the two are combined, it would be easier to read as
```
#if defined(MBEDTLS_ECP_NO_FALLBACK) && defined(MBEDTLS_ECP_NORMALIZE_JAC_ALT)
    return( MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE );
#else
    ...
#endif
```

Reply:
I can do this if it's agreed that the variable declarations can move into the bulk of the code (i.e. after potentially calling the ALT implementation first).

Reply:
Done.